src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
 
-module.exports = sum;
+const PORT = process.env.port || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const filePath = normalizedURL.pathname;
+  const fileName = path.basename(filePath);
+  const fileAbsolutePath = path.join(__dirname, 'public', fileName);
```

Comment:
Regarding your previous issue, you can move your `public` folder one level up and access it here using the same method, but instead of using `__dirname`, try getting the _parent_ directory of `__dirname`.

Hint: there is a method in `path` for that :)